core-bundle/src/Event/JsonLdEvent.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Event;
+
+use Contao\CoreBundle\Routing\ResponseContext\JsonLd\JsonLdManager;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class JsonLdEvent extends Event
```

Comment:
Should we name the `JsonLdEvent` more specific like `JsonLdFinalizeEvent` or `CollectFinalJsonLdEvent`?

Reply:
I named it `JsonLdEvent` because I cannot think of any use case for another JSON-LD related event at all. I don't really like the proposals tbh.